backend/kernelCI_app/management/commands/update_db.py

```
@@ -0,0 +1,533 @@
+from datetime import timedelta
+from django.utils import timezone
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_BATCH_SIZE = 1000
+BUILD_BATCH_SIZE = 10000
+TEST_BATCH_SIZE = 100000
+
+SELECT_BATCH_SIZE = 50000
+
+
+def parse_interval(interval_str: str):
+    now = timezone.now()
+    parts = interval_str.split()
+    if len(parts) != 2:
+        raise ValueError(f"Invalid interval format: {interval_str}")
+
+    value, unit = parts
+    value = int(value)
+
+    if unit.lower() in ["hour", "hours"]:
+        delta = timedelta(hours=value)
+    elif unit.lower() in ["day", "days"]:
+        delta = timedelta(days=value)
+    else:
+        raise ValueError(f"Unsupported time unit: {unit}")
+
+    return now - delta
+
+
+class Command(BaseCommand):
+    help = "Migrate data from default database to dashboard_db"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.start_interval = None
+        self.end_interval = None
+        self.start_timestamp = None
+        self.end_timestamp = None
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--start-interval",
+            type=str,
+            help="Start interval for filtering data ('x days' or 'x hours' format)",
+        )
+        parser.add_argument(
+            "--end-interval",
+            type=str,
+            help="End interval for filtering data ('x days' or 'x hours' format)",
+        )
+        parser.add_argument(
+            "--table",
+            type=str,
+            help="""Table name to limit the migration to
+              (optional, if not provided all tables will be migrated)""",
+        )
+
+    def handle(self, *args, **options):
+        self.start_interval = options.get("start_interval")
+        self.end_interval = options.get("end_interval")
+        table = options.get("table", None)
+
+        if not self.start_interval or not self.end_interval:
```

Should we throw an error when end_interval > start_interval so it avoids empty results?